daffodil-cli/src/it/scala/org/apache/daffodil/udf/TestCLIUdfs.scala

```diff
@@ -588,4 +591,75 @@ class TestCLIUdfs {
       shell.close()
     }
   }
+
+  /**
+   * Tests the case when a UDF class:
+   *    contains a non serializable member
+   */
+  @Test def test_UDFClass_serializability() {
+    val schemaFile = Util.daffodilPath("daffodil-udf/src/test/resources/org/apache/daffodil/udf/genericUdfSchema.xsd")
+    val (testSchemaFile) = if (Util.isWindows) (Util.cmdConvert(schemaFile)) else (schemaFile)
+
+    val metaInfForSomeUdfA = "daffodil-udf/src/test/scala/org/sbadudfs/functionclasses2/StringFunctions/"
+
+    val dafClassPath =
+      (testUdfsPaths :+ Util.daffodilPath(metaInfForSomeUdfA))
+        .mkString(java.io.File.pathSeparator)
+
+    val shell = Util.startIncludeErrors("", envp = Map("DAFFODIL_CLASSPATH" -> dafClassPath))
+
+    try {
+      val cmd = String.format(Util.echoN("strng") + "| %s -v save-parser -s %s -r user_func4", Util.binPath, testSchemaFile)
```

Comment:
The echoN stuff isn't needed to save a parser. Daffodil will just ignore anything from stdin. Should be safe to remove.